python/cudf/cudf/core/column/string.py

```diff
@@ -3673,7 +3673,7 @@ def _memory_usage(self, deep=False):
         if deep:
             return self.__sizeof__()
         else:
-            return self.str().size() * self.dtype.itemsize
+            return len(self) * self.dtype.itemsize
```

Comment:
This is wrong, we should just calculate the correct size and return it always for strings.